capstone/src/main/webapp/script.js

```diff
@@ -214,10 +212,13 @@ function clearMarkers() {
 @param {Object} place location to add to results panel
  */
 function addToDisplayPanel(place) {
-  // put call to get details here and get new place before adding to panel
-  const locationElement = document.getElementById('restaurant-results');
-  locationElement.appendChild(createLocationElement(place));
-  locationElement.appendChild(createAdditionalInfo(place));
+  fetch('/get-details?placeId=' + place.place_id).then((response) =>
+    response.json()).then((place) => {
+    // put call to get details here and get new place before adding to panel
```

Comment:
is this a todo or left over from development?

Reply:
this part is what Carolyn's PR addresses